plugins/transfer/transfer.plugin.zsh

```diff
@@ -0,0 +1,25 @@
+# From https://transfer.sh/
+# Modified by Gabriel Kaam <gabriel@kaam.fr>
+
+function transfer() {
+  if [[ $# -eq 0 ]]; then
```

Comment:
Sorry I didn't see this earlier: the `$basefile` and `$url` variables (and any others meant for this functions internal use) should be made local, so they don't clobber the user's variables. This can be done by sticking `local basefile url` here at the top of the function. Should probably do a `emulate -L zsh` for portability, too.